lib/rucio/rse/rsemanager.py

```diff
@@ -275,12 +285,12 @@ def exists(rse_settings, files, domain='wan', auth_token=None, logger=logging.lo
     ret = {}
     gs = True  # gs represents the global status which inidcates if every operation workd in bulk mode
 
-    protocol = create_protocol(rse_settings, 'read', domain=domain, auth_token=auth_token, logger=logger)
+    protocol = create_protocol(rse_settings, 'read', impl_passed=impl, domain=domain, auth_token=auth_token, logger=logger)
```

Comment:
any particular reason to have this distinction of parameter name: impl/impl_passed?

Reply:
@rcarpa it was giving me errors in the rsemanager file, so i changed it to impl_passed instead of impl there
I can check once more if it throws errors